code/scripting/api/objs/texture.cpp

```diff
@@ -117,6 +117,21 @@ ADE_FUNC(unload, l_Texture, NULL, "Unloads a texture from memory", NULL, NULL)
 	return ADE_RETURN_NIL;
 }
 
+ADE_FUNC(unloadRenderTarget, l_Texture, nullptr, "Unloads a texture's render target. Call this when done drawing to a texture, as it frees up resources.", nullptr, nullptr)
```

Comment:
"unload" is not the right term here since we do not delete any data. Maybe `detachRenderTarget` or simply `destroyRenderTarget`?

Also maybe put into the description that this is needed for cases where a render target is only drawn to once but rendered many times.